aws/kinesis/protobuf/messages.proto

```diff
@@ -1,7 +1,8 @@
+syntax = "proto2";
 import "config.proto";
 
 package aws.kinesis.protobuf;
-option java_package = "com.amazonaws.kinesis.producer.protobuf";
+option java_package = "com.amazonaws.services.kinesis.producer.protobuf";
```

Comment:
why do we need to change this? This was working previously right and we didnt change the path as part of this change

Reply:
this change not relevant to proxy feature but to make it consistent with the java package for Messages.java